Documentation/instruction_name_tables.md

```diff
@@ -0,0 +1,62 @@
+## Database table naming rules
+
+Before uploading a new dataset in the database and creating a new table, the following rules should be followed to name the table:
+
+### All names:
+1. Always in lower case; 
+2. No space between the characters. Instead of using space, use `_` to separate composite names.
+3. Column names should follow the above rules.
```

Comment:
```suggestion
```

As this specification is for all names, maybe you don't need to mention columns

Reply:
I think it's better to let this indication to ensure that the user will not forget it.